api/specs/webserver/openapi.yaml

```diff
@@ -169,6 +168,12 @@ paths:
   /projects/{project_id}/state:
     $ref: "./openapi-projects.yaml#/paths/~1projects~1{project_id}~1state"
 
+  /projects/{project_id}/export:
+    $ref: "./openapi-projects.yaml#/paths/~1projects~1{project_id}~1export"
+
+  /projects:import:
+    $ref: "./openapi-projects.yaml#/paths/~1projects~1import"
+
```

Comment:
it's here you need to have the ```:```. all the others should be using ```/```

see for an example the /projects/{project_id}:open endpoint

Reply:
I'm trying to get this to work as you said but had no success with it.
I've tried to follow the exact same pattern, but had no success with it.
It replies with an error like `uuid:open` dose not exist. It interprets `:open` as part of the parameter.